Add Project Loved: June 2020 news post (#3736)
ppy/osu-wiki: news/2020-06-21-project-loved-june-2020.md

176 BPM, 1:51 | 6.42★, 5.90★, 5.14★  
The \[Hard\] and \[New Mental\] difficulties are *not* being nominated for Loved.

*MC Mental at His Best* stands out as one of the most long-standing mapsets that has spread lighthearted joy among players ever since its submission in 2016. With over 240 favourites already on the mapset, Bullhax's mapset of this silly 176 BPM song features three completed difficulties between 5 and 6 stars. The maps themselves feature a mix of jumps, streams, and finger control, so it's the ultimate mapset to train different skills while simultaneously challenging the ultimate MC Mental.

## <a id="taiko"></a>osu!taiko

osu!taiko Loved candidates were chosen by [jyake](https://osu.ppy.sh/users/9099822) and [-Kazu-](https://osu.ppy.sh/users/920861)!

---

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/taiko/1-amazing-sampling-showcase.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Zekk & SANY-ON - Amazing Sampling Showcase](https://osu.ppy.sh/beatmapsets/1134079#taiko) by [Mew](https://osu.ppy.sh/users/2345156)  
140 BPM, 5:24 | 5.96★  
*description written by [jyake](https://osu.ppy.sh/users/9099822)*

> This map by Mew is the definition of clean, capturing the essence of rhythm game based on drums. No patterns feel out of place; no over-mapping fills the gaps. Although complex streams and crazy technical 1/6 is fun, I would personally like to emphasize how much fun it is to hit notes that feel like part of the music. Mew captures this style of mapping perfectly, with the combination of drum-like mapping and smooth SV that hits just right. This is a very polished map that rewards the player for feeling the beat.

Reviewer: ```suggestion
This map by Mew is the definition of clean, capturing the essence of drum-based rhythm gaming. No patterns feel out of place; no over-mapping fills the gaps. Although complex streams and crazy technical 1/6 is fun, I would personally like to emphasize how much fun it is to hit notes that feel like part of the music. 

Mew captures this style of mapping perfectly, with the combination of drum-like mapping and smooth SV that hits just right. This is a very polished map that rewards the player for feeling the beat.
```

In 2nd place, one of Argentina's newer mappers, juanilp4evr, managed to create a map of truly fine craftsmanship, with slightly wilder SV usage compared to Axer and a more techy approach to the song, leading to a really fun map that truly deserves to be in the top.

Finally, in 1st place, \_MMMMMM_MMMMMM\_ (A.K.A. un-plugged) snatched the win with a creative usage of SV and the clean pattern usage, and he topped it off with his usual gimmicks which have slowly become part of his identity as a mapper. We want to congratulate the winners of the contest for their excellent maps, and also encourage the rest of the community to reach for us in case you too want your contest map featured in the Loved voting!

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/taiko/3-adcar.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Terminal 11 - Adcar](https://osu.ppy.sh/beatmapsets/972116#taiko) by [Ulqui](https://osu.ppy.sh/users/1263669)  
240 BPM, 2:50 | 9.04★  
*description written by [-Kazu-](https://osu.ppy.sh/users/920861)*

> Terminal 11's *Adcar* is an incredibly hectic song with an indescribable feeling that Ulqui somehow managed to materialize in his map: there's fast paced streams, incredibly tricky patterns and a relentless difficulty that will squeeze every drop of your skill if you want to get even close to pass it. And so agreed the folks that select the map pools for osu!taiko World Cup, as it was featured in the Grand Finals of TWC 2020 as a NoMod pick, where the finalists had jaw-dropping performances (and an especially unbelievable performance by applerss, who managed to get an FC against UK). Hopefully, with this addition to the Loved section, we will continue to see incredible scores being made on the leaderboards!

Reviewer: ```suggestion
Terminal 11's *Adcar* is an incredibly hectic song with an indescribable feeling that Ulqui somehow managed to materialize in his map: there's fast paced streams, incredibly tricky patterns and a relentless difficulty that will squeeze every drop of your skill if you want to get even close to pass it. And so agreed the folks that select the map pools for osu!taiko World Cup, as it was featured in the Grand Finals of TWC 2020 as a NoMod pick, where the finalists had jaw-dropping performances (and an especially unbelievable performance by applerss, who managed to get an FC against UK). 

Hopefully, with this addition to the Loved section, we will continue to see incredible scores being made on the leaderboards!
```

[ZYTOKINE - Dancing Dollz feat. cold kiss - REDALiCE Remix](https://osu.ppy.sh/beatmapsets/801848#fruits) by [Rocma](https://osu.ppy.sh/users/566276) et al.  
175 BPM, 3:41 | 5.80★ – 8.49★  
*description written by [Wesley](https://osu.ppy.sh/users/2407265)*

Rocma once again returns to the Loved section, but this time, it is not just Rocma. Rocma has created quite a cool project where not just one or two mappers create a difficulty, but a total of 9 mappers! Because each difficulty is mapped by a different mapper, it gives the whole mapset a nice touch with the variety of patterns all the different difficulties provide. Make sure to give every difficulty a chance, as every difficulty is truly unique!

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/catch/2-kakuzetsu-thanatos.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Rohi - Kakuzetsu Thanatos](https://osu.ppy.sh/beatmapsets/706413#fruits) by [\[\_-Kukkai-\_\]](https://osu.ppy.sh/users/7811952)  
200 BPM, 3:32 | 7.17★  
*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*

> Even though this was mapped long before Rohi became a Featured Artist in osu!, this is a good time to both celebrate the occasion and show the community what a great map \[\_-Kukkai-\_\] made. If you follow osu!catch mapping, then you are probably already familiar with his mapping style, as he is a very active mapper. Compared to his Ranked maps however, his unranked stuff usually goes beyond the ranking criteria and he presents you a lot of patterns that test your abilities. In *Kakuzetsu Thanatos*, he utilized a lot of fast-paced patterns with strong hyperdash usage. Furthermore, there are a handful of patterns that are difficult to keep a high accuracy on due to antiflow and strong hyperdashes which require good catcher control to be caught properly. All in all, this is a fun map which truly deserves more attention.

Reviewer: ```suggestion
Even though this was mapped long before Rohi became a Featured Artist in osu!, this is a good time to both celebrate the occasion and show the community what a great map \[\_-Kukkai-\_\] made. If you follow osu!catch mapping, then you are probably already familiar with his mapping style, as he is a very active mapper. Compared to his Ranked maps however, his unranked stuff usually goes beyond the ranking criteria and he presents you a lot of patterns that test your abilities. 

In *Kakuzetsu Thanatos*, he utilized a lot of fast-paced patterns with strong hyperdash usage. Furthermore, there are a handful of patterns that are difficult to keep a high accuracy on due to antiflow and strong hyperdashes which require good catcher control to be caught properly. All in all, this is a fun map which truly deserves more attention.
```

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/catch/3-yuyushiki-of-nights.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[YuZXs - Yuyushiki of Nights](https://osu.ppy.sh/beatmapsets/238767#fruits) by [Badis](https://osu.ppy.sh/users/559308)  
180 BPM, 2:51 | 5.70★, 5.70★  
*description written by [Sartan](https://osu.ppy.sh/users/4100941)*

> So there's nothing wrong with a little bit of humour every once in a while, right? After about a 14 month wait, we introduce another Badis map to the Loved section: *Yuyushiki of Nights*. This is a fairly dated map now, having been mapped at the end of 2014. Back in those older days, it used to be a frequent multiplayer pick. Despite being old, it doesn't feel all too dated when you play it. The map structure does feel a little weird due to the map accentuating the vocal lines primarily over the instruments; you'll see what I mean when you try it. And try it out you should! It is not the hardest of maps, and the map has two difficulties with different approach rates, making it accessible to most players. So don't let those lovely animated girls wait in the background, give 'em a bit of love too and check out this map!

Reviewer: ```suggestion
So there's nothing wrong with a little bit of humour every once in a while, right? After about a 14 month wait, we introduce another Badis map to the Loved section: *Yuyushiki of Nights*. This is a fairly dated map now, having been mapped at the end of 2014. Back in those older days, it used to be a frequent multiplayer pick. Despite being old, it doesn't feel all too dated when you play it. The map structure does feel a little weird due to the map accentuating the vocal lines primarily over the instruments; you'll see what I mean when you try it. 

And try it out you should! It is not the hardest of maps, and the map has two difficulties with different approach rates, making it accessible to most players. So don't let those lovely animated girls wait in the background, give 'em a bit of love too and check out this map!
```

224 BPM, 4:26 | \[4K\] 4.95★, \[4K\] 5.35★, \[4K\] 5.73★, \[4K\] 6.06★, \[4K\] 4.62★

First in line is a valiant call for salvation. Spare none of an amendment is long-time Loved patron Shoegazer's rebuild of *Over the Fullereneshift*, with cleaner jumpstreams, airtight patterning, and masterful repetition–as with the precision in brand name that Shoegazer tends to carry.

The chart features much difference in approach from the traditional OtFS, with a greater yet subtler focus on musical detail and intricacy. Its cleanliness lends itself nicely to player practice, with its different rates providing adequate coverage as challenge for the many keen-eyed players out there.

Contrary to its name, You Can (sometimes) Redo, and it's clear Shoegazer's call has sufficed.

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/mania/2-gympie-gympe.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[QUIL - Gympie Gympe](https://osu.ppy.sh/beatmapsets/560952#mania) by [Valedict](https://osu.ppy.sh/users/2537082)  
242 BPM, 2:23 | \[4K\], 4.98★ – 6.50★

> Repetitive musical motifs and the subsequent deliberate usage of repetitive patterns seems to be a trend for the first two charts in osu!mania's Loved nominations, with the second being Valedict's *Gympie Gympe*! The chart is a relatively slow jack chart, with the jacks clocking in at 115 BPM, that uses the repeating and developing sounds to push the jacks' density further and further as the chart progresses. This makes playing through not only satisfying, but also allows for the intensity of the play to increase as much as possible until the very end of the chart, making it extremely engaging no matter the skill level of the player. Not only this, but the mapset comes with many extra difficulties with higher rates, making it a great practice chart for people looking to improve their jacks, and a great challenge chart for the top-end players to show off their skills. Ultimately, *Gympie Gympe* became very popular as a result of these things on osu!mania and StepMania/Etterna, and has dug itself a fitting spot among the Loved nominations.

Reviewer: ```suggestion
Repetitive musical motifs and the subsequent deliberate usage of repetitive patterns seems to be a trend for the first two charts in osu!mania's Loved nominations, with the second being Valedict's *Gympie Gympe*! The chart is a relatively slow jack chart, with the jacks clocking in at 115 BPM, that uses the repeating and developing sounds to push the jacks' density further and further as the chart progresses. This makes playing through not only satisfying, but also allows for the intensity of the play to increase as much as possible until the very end of the chart, making it extremely engaging no matter the skill level of the player. 

Not only this, but the mapset comes with many extra difficulties with higher rates, making it a great practice chart for people looking to improve their jacks, and a great challenge chart for the top-end players to show off their skills. Ultimately, *Gympie Gympe* became very popular as a result of these things on osu!mania and StepMania/Etterna, and has dug itself a fitting spot among the Loved nominations.
```

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/mania/3-warriors.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Hommarju - Warriors](https://osu.ppy.sh/beatmapsets/376832#mania) by [ALEFY](https://osu.ppy.sh/users/1291769)  
128 BPM, 2:01 | \[4K\] 3.29★, \[4K\] 3.29★

> Up next is an extremely simple nomination, but one that has gained an immense amount of popularity since it's upload. The map is ALEFY's *Warriors*, a staple of his SV maps that heavily utilizes red line gimmicks. The main draw of the map comes in its choruses, where this usage of red lines is most obvious: instantaneous teleports and 0 BPM lines create a seamless, sudden appearance of following notes and make for an interesting experience similar to that in [maskeara](https://osu.ppy.sh/users/2393914)'s Loved set of *[POSSESSION](https://osu.ppy.sh/beatmapsets/551469)* set. It's no wonder that *Warriors* gained the following it has today given these simple, yet very enjoyable SV gimmicks that keep players coming back. This makes it a clear shoo-in for Loved.

Reviewer: ```suggestion
Up next is an extremely simple nomination, but one that has gained an immense amount of popularity since it's upload. The map is ALEFY's *Warriors*, a staple of his SV maps that heavily utilizes red line gimmicks. The main draw of the map comes in its choruses, where this usage of red lines is most obvious: instantaneous teleports and 0 BPM lines create a seamless, sudden appearance of following notes and make for an interesting experience similar to that in [maskeara](https://osu.ppy.sh/users/2393914)'s Loved set of *[POSSESSION](https://osu.ppy.sh/beatmapsets/551469)* set. 

It's no wonder that *Warriors* gained the following it has today given these simple, yet very enjoyable SV gimmicks that keep players coming back. This makes it a clear shoo-in for Loved.
```

[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/mania/4-chaosmaid.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[paraoka - chaosmaid](https://osu.ppy.sh/beatmapsets/533893#mania) by [shuniki](https://osu.ppy.sh/users/4942174)  
189 BPM, 3:52 | \[4K\] 7.69★, \[4K\] 5.01★

> A familiar face in the loved section, shuniki returns with *chaosmaid*, another fun and challenging long-note community favorite. Like shuniki's other LN maps that were previously nominated for Loved, *chaosmaid* is a staple LN map for all those LN fanatics out there. It's a relatively straightforward map when it comes to shuniki, with its simplistic, repetitive structure executed in a smooth and satisfying manner. At 189 BPM, the map contains dense long-note inverse jumpstream patterns, followed by minijacks and almost rude SV transitions. In the middle section, the gaps between the long notes shrink, making it very necessary for the player to concentrate especially hard, as this section is very difficult to read even in comparison to the inverses. After this, the buildup and chorus repeat themselves, with slightly smaller gaps in the buildup, coming full circle at the end. Overall, it is not surprising that this chart has become so popular among LN players, as it presents a challenge in many different sub-skillsets of LNs. But not only that, the patterns are very smooth and fun, making for an enjoyable experience. With its approachability, popularity, and fun, there's no question that *chaosmaid* deserves a spot in Loved.

Reviewer: ```suggestion
A familiar face in the loved section, shuniki returns with *chaosmaid*, another fun and challenging long-note community favorite. Like shuniki's other LN maps that were previously nominated for Loved, *chaosmaid* is a staple LN map for all those LN fanatics out there. It's a relatively straightforward map when it comes to shuniki, with its simplistic, repetitive structure executed in a smooth and satisfying manner. 

At 189 BPM, the map contains dense long-note inverse jumpstream patterns, followed by minijacks and almost rude SV transitions. In the middle section, the gaps between the long notes shrink, making it very necessary for the player to concentrate especially hard, as this section is very difficult to read even in comparison to the inverses. After this, the buildup and chorus repeat themselves, with slightly smaller gaps in the buildup, coming full circle at the end. 

Overall, it is not surprising that this chart has become so popular among LN players, as it presents a challenge in many different sub-skillsets of LNs. But not only that, the patterns are very smooth and fun, making for an enjoyable experience. With its approachability, popularity, and fun, there's no question that *chaosmaid* deserves a spot in Loved.
```
